docs/en/writing-running-appium/tutorial/swipe/simple-element.md

```diff
@@ -0,0 +1,206 @@
+## Simple Element swipe
+
+### Android: UIAutomator2
+
+With UIAutomator2 driver, by default, we see only elements on screen. If
+element continues outside screen, then this part will be hidden.
```

Comment:
if an elements container covers the area inside and outside of the current view port then a part of its content won't be visible